app/controllers/idv/image_uploads_controller.rb

```diff
@@ -21,7 +21,11 @@ def create
 
         update_analytics(client_response)
 
-        store_pii(client_response) if client_response.success?
+        if client_response.success?
+          doc_pii_form_response = Idv::DocPiiForm.new(client_response.pii_from_doc).submit
+          form_response = form_response.merge(doc_pii_form_response)
```

Comment:
we need an `analytics.track_event` after this merge, or potentially move the one we have down after this block

Reply:
Is it ok to add under the same analytics key but with an updated value, or should a new one be created?

Reply:
"updated value" -- are you asking if it's OK to add new data in to the existing event by moving that call down lower in the action? Yes, that should totally be fine

We should not call the same event name twice within the same request, we'd have no good way to tell them apart. If we did *need* to have separate events, we could add a new event name, those are free

Reply:
got it, moved the analytics tracking to below after it's potentially been merged with the PII validation form